### PR TITLE
UX: Update topics stats automatically 

### PR DESCRIPTION
**Pull request by megothss**
Updates automatically data on the stats section of the topic.

It will update automatically the following information: likes, replies and last reply (timestamp and user)

image

### CODE REVIEW

**Review by pmusaraj — COMMENTED** [review]
This looks good to me, left a few very minor comments around tests.

File: app/models/post.rb [hunk 1]
```diff
@@ -198,6 +198,9 @@ def publish_change_to_clients!(type, opts = {})
     # but message is safe to skip
     return unless topic
 
+    skip_topic_stats = opts[:skip_topic_stats]
```
Comment (romanrizzi) [inline app/models/post.rb:201]:
```suggestion
    skip_topic_stats = opts.delete(:skip_topic_stats)
```

File: spec/lib/post_action_creator_spec.rb [hunk 2]
```diff
@@ -67,12 +67,31 @@
...
+    it 'notifies updated topic stats to subscribers' do
+      topic = Fabricate(:topic)
+      post = Fabricate(:post, topic: topic)
+
+      expect(post.reload.like_count).to eq(0)
+
+      MessageBus.expects(:publish).at_least_once
+
+      # tests if messages of type :stats are published and the like_count is fetched from the topic
+      MessageBus.expects(:publish).once.with do |channel, message, _|
```
Comment (romanrizzi) [inline spec/lib/post_action_creator_spec.rb:86]: MessageBus has a method called `track_publish`, which you can use see the contents of a message.

Comment (megothss) [reply spec/lib/post_action_creator_spec.rb]: Done!

**Pull request merged**